Cargo.toml

```diff
@@ -36,6 +36,8 @@ url = "2.2.2"
 validator = { version = "0.14", features = ["derive"] }
 walrus = "0.19.0"
 wasmparser = "0.83.0"
+oci-distribution = { git = "https://github.com/krustlet/oci-distribution", rev = "0f717968093a5415f428503d741dedf24ea97948", default-features = false, features =  ["rustls-tls"] }
```

Comment:
AFAIK this is being exposed by `policy-fetcher`. Try to consume it through the export done by `policy-fetcher`

Reply:
it is used in `policy-fetcher` but it is not publicly exposed. kwctl does not have a direct dependency of `policy-fetcher`. 
I can export oci-distribution in `policy-fetcher` and then in `policy-evaluator`. Would this be ok? or am I missing something?
if this is ok I'll submit PRs for `policy-fetcher` and `policy-evaluator`